google_symptoms/tests/test_data/state_data_20220916-20220924.csv

```diff
@@ -0,0 +1,19 @@
+open_covid_region_code,date,symptom_Cough,symptom_Phlegm,symptom_Sputum,symptom_Upper_respiratory_tract_infection,symptom_Nasal_congestion,symptom_Post_nasal_drip,symptom_Rhinorrhea,symptom_Sinusitis,symptom_Rhinitis,symptom_Common_cold,symptom_Fever,symptom_Hyperthermia,symptom_Chills,symptom_Shivering,symptom_Low_grade_fever,symptom_Shortness_of_breath,symptom_Wheeze,symptom_Croup,symptom_Pneumonia,symptom_Asthma,symptom_Crackles,symptom_Acute_bronchitis,symptom_Bronchitis,symptom_Anosmia,symptom_Dysgeusia,symptom_Ageusia,symptom_Laryngitis,symptom_Sore_throat,symptom_Throat_irritation,symptom_Type_2_diabetes,symptom_Urinary_tract_infection,symptom_Hair_loss,symptom_Candidiasis,symptom_Weight_gain
```

Comment:
was this change intentional?

Reply:
I added this file for the new test. It has the same format as existing test files but is smaller so the test runs faster. For the sake of keeping test data to a minimum, switching this to use an existing test file.